source/docs/yearly-overview/known-issues.rst

```diff
@@ -169,3 +153,28 @@ Issues with WPILib Dashboards and Simulation on Windows N Editions
 
 **Solution:** Install the `Media Feature Pack <https://www.microsoft.com/en-us/software-download/mediafeaturepack>`__
 
+Fixed in WPILib 2024.2.1
+------------------------
+
+Visual Studio Code Reports Unresolved Dependency
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Issue:** Java programs will report ``Unresolved dependency: org.junit.platform junit-platform-launcherJava(0)`` on build.gradle. Programs that use unit tests will fail to build. This causes build.gradle to be highlighted red in the Visual Studio Code explorer, and the ``plugins`` line in build.gradle to have a red squiggle.
+
+**Workaround:** This can be safetly ignored if you aren't running unit tests. To fix it, do the following:
```

Comment:
I'd change this to state the fix is to update WPILib, and that the below instructions were how to fix it manually.

Reply:
It's in the heading of fixed in 2024.2.1, like we've done other years.